standard/flags.lua

```diff
@@ -62,6 +62,54 @@ function Flags.Icon(args, flagName)
 	end
 end
 
+-- Returns the localisation/country-adjective of a country or region
+--[[
+supported args are:
+country			- country name, flag code, or alias of the Flag
+displayNoError		- boolean that decides if there should be a displayed error if no entry is found
+shouldReturnSimpleError	- boolean that decides if displayed error should be simple or detailed
```

Comment:
since (longterm) we want to replace https://liquipedia.net/commons/Module:Localisation with this PR i added the same options that existed there
if we change the var names we would have to change more stuff when replacing the old module

not necessarily opposed to it, just mentioning it^^

Reply:
Since we would need to move the usages over one by one anyway, this probably the best opportunity to refactor the code

Reply:
done